backend/src/index.ts

```diff
@@ -18,6 +18,13 @@ app.use(express.json());
 app.use(cors());
 app.use(
   session({
+    // TODO: uncomment the secure 
+    // secure: true,
```

Comment:
i recommend we instead use `process.env.NODE_ENV` to determine whether we're in production/development and toggle this flag accordingly so we don't forget abt it

Reply:
would this be using another env variable like DEV_STATUS or is there already somethin we can use?